src/main/java/com/authzed/util/ClientToken.java

```diff
@@ -0,0 +1,40 @@
+package com.authzed.util;
```

Comment:
can we make this `grpcutil`? since that's what we've called it in all of the other client libraries